src/radtest/radtest.c

```diff
@@ -51,7 +51,8 @@ void rtst_main (void){
 
 		if ((RtstTickCnt % RTST_HEARTBEAT_TICKS) == 0) {
 			// Make heartbeat every 10 sec.
-			SysEvent(RTST_MODNR, EVENT_WARNING, RTST_EVENTID_HEARTBEAT, "Supervision watchdog feed\n", 26);
+			//SysEvent(RTST_MODNR, EVENT_WARNING, RTST_EVENTID_HEARTBEAT, "Supervision watchdog feed\n", 26);
+			SysEvent_Debug("Supervision watchdog feed\n", 26)
```

Comment:
I tried to make a function that prints pure strin without id message

Which is latter to be converted to pure binary string. 

![image](https://user-images.githubusercontent.com/66519643/183976531-3716dae7-e1bd-40f4-a8f3-137e29da5666.png)

I still have some ~ new line symbols comming from somewhere.

Please take a look. 

If no - we can revert this commit